gcp_monitoring/README.md

```diff
@@ -0,0 +1,46 @@
+# Google Cloud Platform Monitoring Template
+
+Provided by: [bonitoo.io](.)
+
+**Display data from Google Cloud Monitoring -- formerly Stackdriver -- using the
+Cloud Monitoring API v3.**
+
+1. MySQL database hosted in Google Cloud SQL
+2. The Compute dashboard provides deeper insight into the GCP compute services
+   provided by GCP Stackdriver.
+3. HTTPS LoadBalancing dashboard monitors network load balancing in the cluster.
```

Comment:
Are these things included in the template? Requirements? etc. Need some context for what this list is.

Reply:
Three dashboards